src/Dialog/index.scss

```diff
@@ -1,3 +1,7 @@
+.nds-dialog {
+  --overflow-gradient-height: 20px;
```

Comment:
CSS Custom Properties cascade, so here I'm setting a var that only works within the children of `nds-dialog`, which prevents polluting our nice list of global vars

Reply:
🧠